webpack/ForemanTasks/Components/TasksTable/TasksTableReducer.js

```diff
@@ -0,0 +1,53 @@
+import Immutable from 'seamless-immutable';
+import { combineReducers } from 'redux';
+import { createTableReducer } from 'foremanReact/components/common/table';
+import {
+  TASKS_CONTROLLER,
+  TASKS_SUCCESS,
+  SET_SORT_TASKS_TABLE,
+  SET_PAGINATION_TASKS_TABLE,
+} from './TasksTableConstants';
+
+const initialState = Immutable({
+  query: null,
+  itemCount: 0,
+  pagination: {
+    page: 1,
+    perPage: 20,
+  },
+});
+
+export const TasksTableQueryReducer = (state = initialState, action) => {
+  const { type, payload } = action;
+  switch (type) {
+    case TASKS_SUCCESS:
+      return Immutable.merge(state, {
+        itemCount: payload.subtotal,
+        pagination: {
+          page: Number(payload.page),
+          perPage: Number(payload.per_page),
```

Comment:
Can you refactor it into a number in the action that fires this instead?

Reply:
Nope, this is how we get it from the api and the action is in foreman (not foreman tasks)

Reply:
Debugging it both in Foreman and here, I see it is returning a number.

Reply:
but when clicking on pagination I see it changes into a string.. weird
anyway it is non blocking for this PR.